docker-compose-test.yaml

```diff
@@ -0,0 +1,65 @@
+version: '3.3'
+
+services:
+  graphdb:
+    image: docker-registry.ontotext.com/graphdb:${GDB_VERSION}
+    environment:
+      GDB_JAVA_OPTS: >-
+        -Dgraphdb.workbench.importDirectory=/opt/home/import-data/
+        -Dgraphdb.jsonld.whitelist=https://w3c.github.io/json-ld-api/tests/*
+        -Dgraphdb.stats.default=disabled
+        -Dgraphdb.foreground=
+        -Dgraphdb.logger.root.level=ERROR
+    ports:
+      - '7200:7200'
+    volumes:
+      - ./e2e-tests/fixtures/graphdb-import:/opt/home/import-data/
+    networks:
+      - single-spa-network
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f -H 'Accept: text/html' ${GRAPHDB_URL}/index.html"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  workbench:
+    build:
+      context: .
+      dockerfile: root-config.Dockerfile
+    environment:
+      GRAPHDB_URL: ${GRAPHDB_URL}
+      WORKBENCH_PORT: ${WORKBENCH_PORT}
+    depends_on:
+      - graphdb
+    ports:
+      - ${WORKBENCH_PORT}
+    networks:
+      - single-spa-network
+    healthcheck:
+      test: ["CMD-SHELL", "apk add --no-cache curl && curl -f -H 'Accept: */*' http://workbench:${WORKBENCH_PORT}"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  cypress:
+    image: cypress/included
```

Comment:
Should we depend on concrete version here? We don't wan't undeterministic behavior when running tests, do we?

Reply:
fixed